pmm-admin.go

```diff
@@ -252,7 +252,7 @@ Table statistics is automatically disabled when there are more than 10000 tables
 		},
 	}
 	cmdAddLinuxMetrics = &cobra.Command{
-		Use:   "linux:metrics [name] [-- exporter_args]",
+		Use:   "linux:metrics [name] [flags] [-- exporter_args]",
```

Comment:
It works but I don't know well Cobra, not sure if this is the best way to fix the description.

Reply:
Are flags related to the 'pmm-admin add' command or to the *linux:metrics* monitoring service? Currently, the documentation reads this way (although I would replace SERVICE with MONITORING_SERVICE):
**pmm-admin add [OPTIONS] [SERVICE]**

Reply:
That line is for `pmm-admin add linux:metrics` only.

Reply:
As you have on the screenshot in the description there is currently a message:
```
Usage:
    pmm-admin add linux:metrics [name] [-- exporter_args] [flags]
```
And of course I've made a mistake:/ Fixed order:
```
Usage:
    pmm-admin add linux:metrics [flags] [name] [-- exporter_args]
```

I think that's the correct order. Having said that, it really doesn't matter where you pass flags, you can have something like 
```
pmm-admin --flag1 add --flag2 linux:metrics --flag3 name_xyz -- exporter-arg
```
and will work, as long as the additional exporter args are at the end, after double dash.

Still though, I believe documentation should suggest placing `[flags]` just after commands.